runelite-api/src/main/java/net/runelite/api/events/player/headicon/OverheadPrayerChanged.java

```diff
@@ -0,0 +1,21 @@
+package net.runelite.api.events.player.headicon;
+
+import lombok.Getter;
+import net.runelite.api.HeadIcon;
+import net.runelite.api.Player;
+
+public class OverheadPrayerChanged extends PlayerHeadIconChanged
+{
+	@Getter
+	private final HeadIcon oldHeadIcon;
+
+	@Getter
+	private final HeadIcon newHeadIcon;
+
+	public OverheadPrayerChanged(Player player, HeadIcon changedFrom, HeadIcon changedTo)
```

Comment:
change Player -> Actor

Reply:
the event is only produced from the RSPlayer class atm cuz I didnt wanna half to deal with instanceof checks when handling it but i now realize that is useful for npcs too.

should i move it to RSActor or add a separate event for NPcs?